demos/common/cpp/models/include/models/image_model.h

```diff
@@ -43,4 +44,6 @@ class ImageModel : public ModelBase {
 
     size_t netInputHeight = 0;
     size_t netInputWidth = 0;
+    INTERPOLATION_MODE interpolationMode = LINEAR;
+    RESIZE_MODE resizeMode = RESIZE_FILL;
```

Comment:
What's the reason to introduce these fields? I think each of the wrappers could hard code them in a place where `resizeImageExt()` is called.

Reply:
Sometimes we need to use image_model preprocessing to avoid duplication of code. For these cases parameters are useful